tests/cli/test_push.py

```diff
@@ -1,11 +1,13 @@
 import pytest
 import sqlalchemy as sa
+from spinta.core.config import RawConfig
 from spinta.manifests.tabular.helpers import striptable
 from spinta.testing.cli import SpintaCliRunner
 from spinta.testing.data import listdata
 from spinta.testing.datasets import create_sqlite_db
 from spinta.testing.tabular import create_tabular_manifest
-from tests.datasets.test_sql import configure_remote_server, create_rc
+from tests.datasets.test_sql import configure_remote_server, create_rc, create_client
```

Comment:
Do not import functions from other tests. If you need to do that, move function into `spinta.testing`.